airbyte-integrations/connectors/source-mailchimp/integration_tests/expected_records.txt

```diff
@@ -0,0 +1,25 @@
settings": {"subject_line": "Weekly Bytes | Learnings from Soft Launch and our Vision", "preview_text": "Here are Airbyte's community news and product updates of the past week.\u00a0What We Learned from Our Soft Launch.", "title": "Newsletter 10/08", "from_name": "Airbyte Team", "reply_to": "hey@airbyte.io", "use_conversation": false, "to_name": "", "folder_id": "", "authenticate": true, "auto_footer": false, "inline_css": false, "auto_tweet": false, "fb_comments": true, "timewarp": false, "template_id": 394434, "drag_and_drop": false}, "tracking": {"opens": true, "html_clicks": true, "text_clicks": false, "goal_tracking": false, "ecomm360": false, "google_analytics": "", "clicktale": ""}, "report_summary": {"opens": 71, "unique_opens": 45, "open_rate": 0.8035714285714286, "clicks": 11, "subscriber_clicks": 4, "click_rate": 0.07142857142857142, "ecommerce": {"total_orders": 0, "total_spent": 0, "total_revenue": 0}}, "delivery_status": {"enabled": false}, "_links": [{"rel": "parent", "href": "https://us2.api.mailchimp.com/3.0/campaigns", "method": "GET", "targetSchema": "https://us2.api.mailchimp.com/schema/3.0/Definitions/Campaigns/CollectionResponse.json", "schema": "https://us2.api.mailchimp.com/schema/3.0/Paths/Campaigns/Collection.json"}, {"rel": "self", "href": "https://us2.api.mailchimp.com/3.0/campaigns/49d68626f3", "method": "GET", "targetSchema": "https://us2.api.mailchimp.com/schema/3.0/Definitions/Campaigns/Response.json"}, {"rel": "delete", "href": "https://us2.api.mailchimp.com/3.0/campaigns/49d68626f3", "method": "DELETE"}, {"rel": "send", "href": "https://us2.api.mailchimp.com/3.0/campaigns/49d68626f3/actions/send", "method": "POST"}, {"rel": "cancel_send", "href": "https://us2.api.mailchimp.com/3.0/campaigns/49d68626f3/actions/cancel-send", "method": "POST"}, {"rel": "feedback", "href": "https://us2.api.mailchimp.com/3.0/campaigns/49d68626f3/feedback", "method": "GET", "targetSchema": "https://us2.api.mailchimp.com/schema/3.0/Definitions/Campaigns/Feedback/CollectionResponse.json"}, {"rel": "content", "href": "https://us2.api.mailchimp.com/3.0/campaigns/49d68626f3/content", "method": "GET", "targetSchema": "https://us2.api.mailchimp.com/schema/3.0/Definitions/Campaigns/Content/Response.json"}, {"rel": "send_checklist", "href": "https://us2.api.mailchimp.com/3.0/campaigns/49d68626f3/send-checklist", "method": "GET", "targetSchema": "https://us2.api.mailchimp.com/schema/3.0/Definitions/Campaigns/Checklist/Response.json"}, {"rel": "pause", "href": "https://us2.api.mailchimp.com/3.0/campaigns/49d68626f3/actions/pause", "method": "POST"}, {"rel": "resume", "href": "https://us2.api.mailchimp.com/3.0/campaigns/49d68626f3/actions/resume", "method": "POST"}, {"rel": "replicate", "href": "https://us2.api.mailchimp.com/3.0/campaigns/49d68626f3/actions/replicate", "method": "POST"}, {"rel": "create_resend", "href": "https://us2.api.mailchimp.com/3.0/campaigns/49d68626f3/actions/create-resend", "method": "POST"}]}, "emitted_at": 1671548438209}
```

Comment:
This is likely disclosing internal mailchimp data. Please:
- erase it
- log an issue to set up a sandbox Mailchimp account
- bypass expected_records test

Reply:
thanks for flagging. done

Reply:
@alafanechere we have testing account.Just want to figure out what internal data are we disclosing.

Reply:
The data sample looks like past emailing campaigns results with email addresses we don't want to disclose.